diff -up pim-sieve-editor-19.04.2/src/org.kde.sieveeditor.desktop.1~ pim-sieve-editor-19.04.2/src/org.kde.sieveeditor.desktop
--- pim-sieve-editor-19.04.2/src/org.kde.sieveeditor.desktop.1~	2019-05-21 08:45:18.000000000 +0200
+++ pim-sieve-editor-19.04.2/src/org.kde.sieveeditor.desktop	2019-06-15 02:12:15.875037077 +0200
@@ -110,7 +110,7 @@ Comment[x-test]=xxEdit IMAP Sieve filter
 Comment[zh_CN]=编辑 IMAP 筛选过滤器
 Comment[zh_TW]=編輯 IMAP 篩選器
 Terminal=false
-Categories=Qt;KDE;Office;Network;X-KDE-Sieve;
+Categories=Qt;KDE;Office;Network;X-KDE-Sieve;X-KDE-More;
 InitialPreference=10
 X-DBUS-StartupType=Unique
 X-DBUS-ServiceName=org.kde.sieveeditor
